### ISSUE TITLE
[bug]: update js-controlle brings error

### ISSUE THREAD

**Issue by BenAhrdt** [ioBroker/ioBroker.js-controller#2618]
### No existing issues.

- [X] There is no existing issue for my problem.

### Description

today i update the js-controller from 5.0.18 to5 .0.19.

this message comes up.

![image](https://github.com/ioBroker/ioBroker.js-controller/assets/92173565/d29ac874-4665-4338-9ead-d46c82c57bdb)

i hope everything works. after i click close the js-controller is up to 5.0.19

**Comment by foxriver76** [comment]
yes all fine, just an issue in communication from browser with the server, if so it is an admin issue, as it might determined the wrong address of the backend.
